docker-compose.yml

```diff
@@ -20,3 +20,17 @@ services:
       - "1234:1234"
     networks:
       - main-network
+
+  cimgruby:
```

Comment:
Depending on what the solution ends up being I'm not sure we need this to be defined here. It's entirely appropriate to run the tests from your host machine. I could see this being useful if we end up making use of this within our circleci execution though and so am open to keeping it around.